packages/utils-node/package.json

```diff
@@ -24,6 +24,7 @@
     "commit": "concurrently \"yarn test --coverage --no-cache --silent --forceExit --detectOpenHandles --runInBand --watch=false && jest-coverage-badges --input src/tests/coverage/coverage-summary.json --output src/tests/badges\" \"yarn lint\""
   },
   "dependencies": {
+    "@homeservenow/serverless-aws-handler": "^0.2.4",
```

Comment:
what do you need this for in here?

Reply:
it's used here https://github.com/reapit/foundations/blob/master/packages/utils-node/src/authorisation/authorised.ts#L1 no idea why this was able to build before 😂